lib/school.rb

```diff
@@ -0,0 +1,60 @@
+class School
+  attr_reader :start_time,
+              :hours_in_school_day,
+              :student_names
+  def initialize(start_time, hours_in_school_day)
+    @start_time          = start_time
+    @hours_in_school_day = hours_in_school_day
+    @student_names       = []
+  end
+
+  def add_student_name(name)
+    @student_names << name
+  end
+
+  def military_time
+    start_time[0].to_i + hours_in_school_day
+  end
+
+  def end_time
+    military_time.to_s + ":00"
+  end
+
+  def is_full_time?
+    if @hours_in_school_day >= 4
+      true
+    else
+      false
+    end
+  end
```

Comment:
You could also rely on the return value of the conditional statement here:

```ruby
def is_full_time?
  @hours_in_school_day >= 4
end
```

Reply:
@memcmahon Thank you!